pmg/templates/attendance_overview.html

```diff
@@ -13,98 +13,56 @@ <h1><i class="fa fa-fw fa-table" aria-hidden="true"></i> Committee meeting atten
   <a class="btn btn-sm btn-info" href="{{ url_for('archived_attendance_overview')}}">Historical Attendance</a>
 
   <h2 id="national-assembly">National Assembly</h2>
-  {% if not attendance %}
+  {% if not attendance_na %}
   <h4>Data for current parliament is not avaliable yet.</h4>
   {% else %}
   <table class="table table-hover sortable">
                               <thead>
-	                          <th class="th-rank"></th>
-	                          <th class="th-cte-name">Committee</th>
-	                          <th class="th-number-meetings hidden-xs" data-firstsort="desc">Meetings</th>
-	                          <th class="th-attendance" data-defaultsort="desc">Attendance</th>
-	                          <th class="th-attendance-change">Since 2019</th>
+                                  <th class="th-rank"></th>
+                                  <th class="th-cte-name">Committee</th>
+                                  <th class="th-number-meetings hidden-xs" data-firstsort="desc">Meetings</th>
+                                  <th class="th-attendance" data-defaultsort="desc">Attendance</th>
       </thead>
                               {% for item in attendance_na %}
                               <tr id="{{ item.committee_id }}">
-	                          <td class="rank">#{{ loop.index }}</td>
-	                          <td class="cte-name"><a href="{{ url_for('committee_detail', committee_id=item.committee_id) }}">{{ item.committee }}</a></td>
-	                          <td class="number-meetings hidden-xs">{{ item.n_meetings }}</td>
-	                          <td class="attendance" data-value="{{ item.avg_attendance }}">
+                                  <td class="rank">#{{ loop.index }}</td>
+                                  <td class="cte-name"><a href="{{ url_for('committee_detail', committee_id=item.committee_id) }}">{{ item.committee }}</a></td>
+                                  <td class="number-meetings hidden-xs">{{ item.n_meetings }}</td>
+                                  <td class="attendance" data-value="{{ item.avg_attendance }}">
                                       <div class="progress">
-		                          <div class="progress-bar progress-bar-success" role="progressbar" aria-valuenow="{{ item.avg_attendance }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ item.avg_attendance }}%;">
-		                              {{ item.avg_attendance|round|int }}%
-		  </div>
+                                          <div class="progress-bar progress-bar-success" role="progressbar" aria-valuenow="{{ item.avg_attendance }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ item.avg_attendance }}%;">
+                                              {{ item.avg_attendance|round|int }}%
+                  </div>
               </div>
-	  </td>
-	                          <td class="attendance-change" data-value="{{ item.change }}">
```

Comment:
I don't think the solution is to remove a feature that has to be re-added before the end of the year.
I think the solution is to add a test so we know it works, and then add a condition to hide the column when the information isn't available yet.

If that's hard because the backend is currently hardcoding cut-off dates, you can add a template-only test with mocked context data.

Reply:
After looking at this again. I think this is the way to go.
We will be making a change(hiding the 2019 column) that effects this page once every 5 years for a few months.
To get that column back would involve ideally a git revert.
Thoughts?

Reply:
The reason I'm suggesting we add a test and dynamically hide this is that ongoing maintenance is amost guaranteed to mean that it won't be as simple as a git revert.

By keeping the feature in the page and having a test, we're forced to keep it in mind with ongoing maintenance.

Most importantly, it's working right now. Including a test of this should take like an hour.
Fixing it when it turns out it's not just a git revert will take more than an hour. and either means we have to remember to check for it coming up to next year, or it will mean stress because it turns out not to work.

If testing for it working and making it work is so hard, it suggests there's a problem with our ability to test for stuff, which needs to be fixed anyway.